ui/utils.ts

```diff
@@ -40,36 +38,14 @@ let client_auditor: Timer;
  */
 function periodic_client_query() {
     // for now only query when was the last time balance data was saved
-    service.query_last_balance_save_time().then(value => {
-        settings.last_balance_save = value;
+    service.query_periodic_data().then(result => {
+        settings.last_balance_save = result['last_balance_save'];
+        update_eth_node_connection_status_ui(result['eth_node_connection']);
     }).catch(reason => {
         console.log('Error at periodic client query' + reason);
```

Comment:
It would be better to call `showError` here.

Reply:
Thanks. Done.